Resources/public/js/models/ez-trashitemmodel.js

```diff
@@ -39,17 +39,22 @@ YUI.add('ez-trashitemmodel', function (Y) {
         },
 
         /**
-         * Restores the item to it's original location
+         * Restores the item to it's original location or to a given one if specified
          *
          * @method restore
          * @param {Object} options the required for the update
          * @param {Object} options.api (required) the JS REST client instance
+         * @param {Object} options.destination (optional) locationId under which the item will be restored.
```

Comment:
instead of writing `@param {Object} options.destination (optional)`, write `@param {Object} [options.destination]`